src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
I have no issue at all with including syntactic sugar.

But I agree here.  This is not a common enough case to warrant dedicated sugar.